components/mediation/inbound-endpoints/org.wso2.micro.integrator.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/internal/http/api/InternalAPIDispatcher.java

```diff
@@ -129,7 +129,8 @@ private APIResource findResource(MessageContext synCtx, InternalAPI internalApi)
     }
 
     private String getSupportedMethodsForInternalApis() {
-        return "GET, POST, PUT, DELETE, OPTIONS";
+
+        return RESTConstants.REST_ALL_SUPPORTED_METHODS + "OPTIONS, PATCH";
```

Comment:
Do we need a comma before  OPTIONS, PATCH?

Reply:
yes we do , fixed it . thanks .